### PR TITLE
refactor!: remove deprecated editor, TUI_EDITOR_STYLES and styles constants

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: projects/cdk/schematics/ng-update/constants/warnings.ts [hunk 1]
```diff
@@ -106,4 +106,10 @@ export const MIGRATION_WARNINGS: readonly MigrationWarning[] = [
         moduleSpecifier: '@taiga-ui/core',
         message: 'colorFallback has been deleted in 3.0, please use CSS variables',
     },
+    {
```
Comment (splincode) [inline projects/cdk/schematics/ng-update/constants/warnings.ts:109]: we can add this to main branch

File: projects/demo/src/modules/components/editor/examples/import/import-module.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-```ts
+```typescript
```
Comment (splincode) [inline projects/demo/src/modules/components/editor/examples/import/import-module.md:1]: ts

File: projects/cdk/schematics/ng-update/constants/warnings.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/cdk/schematics/ng-update/constants/warnings.ts:109]: Hmm.. It's breaking change, we can't
if you mean migration only, that doesn't make sense in main branch

Comment (splincode) [reply projects/cdk/schematics/ng-update/constants/warnings.ts:109]: oh, sorry

**Review by splincode — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

File: projects/addon-editor/components/editor-socket/editor-socket.component.ts [hunk 3]
```diff
@@ -11,12 +10,11 @@ import {
     SecurityContext,
     ViewEncapsulation,
 } from '@angular/core';
-import {TUI_EDITOR_STYLES} from '@taiga-ui/addon-editor/tokens';
 import {TUI_SANITIZER} from '@taiga-ui/core';
 
 // @dynamic
 @Component({
-    selector: 'tui-editor-socket',
+    selector: 'tui-editor-socket,[tuiEditorSocket]',
```
Comment (waterplea) [inline projects/addon-editor/components/editor-socket/editor-socket.component.ts:17]: Do we really need two selectors? TipTap cannot work with custom tag as host?

File: projects/addon-editor/components/editor/editor.style.less [hunk 4]
```diff
@@ -4,53 +4,44 @@
...
     display: block;
     font: var(--tui-font-text-m);
-}
 
-.t-outline {
-    .fullsize();
-    border-radius: var(--tui-radius-m);
-    pointer-events: none;
-}
+    & ::ng-deep .tui-editor-socket {
```
Comment (waterplea) [inline projects/addon-editor/components/editor/editor.style.less:8]: Why ng-deep?

File: projects/demo/src/modules/components/editor/editor.component.ts [hunk 5]
```diff
@@ -14,20 +19,74 @@ import {ABSTRACT_PROPS_ACCESSOR} from '../abstract/inherited-documentation/abstr
     providers: [
         {
             provide: ABSTRACT_PROPS_ACCESSOR,
-            useExisting: forwardRef(() => ExampleTuiEditorComponent),
+            useExisting: forwardRef(() => ExampleEditorNewComponent),
```
Comment (waterplea) [inline projects/demo/src/modules/components/editor/editor.component.ts:22]: We should probably drop `New` from example classes

**Review by waterplea — APPROVED** [review]
Just a few small concerns

File: projects/addon-editor/components/editor-socket/editor-socket.component.ts (hunk 3 above)
Comment (vladimirpotekhin) [reply projects/addon-editor/components/editor-socket/editor-socket.component.ts]: removed

**Pull request merged**